### PR TITLE
Updates outdated tutorial and example query files

### LINKED ISSUE #413: Fix invalid PartiQL tutorial queries
The [PartiQL tutorial](https://partiql.org/tutorial.html) has a broken query in the "Unnesting data with LEFT JOIN always preserves parent information" [section](https://partiql.org/tutorial.html#unnesting-data-with-left-join-always-preserves-parent-information). 

Reproduction using the v0.2.6 cli:
```
❯ ./bin/partiql  -e Tutorial/code/tutorial-all-data.env
Welcome to the PartiQL REPL!
Using version: 0.2.6-f050c87
PartiQL> SELECT e.id AS id, e.name AS employeeName, e.title AS title, p.name AS projectName FROM hr.employeesNest AS e LEFT JOIN e.projects AS p;
org.partiql.lang.syntax.ParserException: Expected 'ON'
	Parser Error: at line 1, column 136: malformed use of FROM with JOIN, SEMICOLON : ';'

	at org.partiql.lang.util.TokenListExtensionsKt.err(TokenListExtensions.kt:59)
	at org.partiql.lang.util.TokenListExtensionsKt.err(TokenListExtensions.kt:74)
	at org.partiql.lang.util.TokenListExtensionsKt.err$default(TokenListExtensions.kt:73)
	at org.partiql.lang.syntax.SqlParser.parseFromSource(SqlParser.kt:2082)
	at org.partiql.lang.syntax.SqlParser.parseFromSource$default(SqlParser.kt:2000)
	at org.partiql.lang.syntax.SqlParser.parseFromSourceList(SqlParser.kt:2115)
	at org.partiql.lang.syntax.SqlParser.parseSelectAfterProjection(SqlParser.kt:1559)
	at org.partiql.lang.syntax.SqlParser.parseSelect(SqlParser.kt:1333)
	at org.partiql.lang.syntax.SqlParser.parseTerm(SqlParser.kt:1006)
	at org.partiql.lang.syntax.SqlParser.parsePathTerm(SqlParser.kt:920)
	at org.partiql.lang.syntax.SqlParser.parsePathTerm$default(SqlParser.kt:918)
	at org.partiql.lang.syntax.SqlParser.parseUnaryTerm(SqlParser.kt:914)
	at org.partiql.lang.syntax.SqlParser.parseExpression$lang(SqlParser.kt:801)
	at org.partiql.lang.syntax.SqlParser.parseExpression$lang$default(SqlParser.kt:800)
	at org.partiql.lang.syntax.SqlParser.parseExprNode(SqlParser.kt:2266)
	at org.partiql.lang.CompilerPipelineImpl.compile(CompilerPipeline.kt:195)
	at org.partiql.cli.Repl$executePartiQL$1.invoke(Repl.kt:258)
	at org.partiql.cli.Repl$executePartiQL$1.invoke(Repl.kt:107)
	at org.partiql.cli.Repl.executeTemplate(Repl.kt:223)
	at org.partiql.cli.Repl.executePartiQL(Repl.kt:254)
	at org.partiql.cli.Repl.run(Repl.kt:323)
	at org.partiql.cli.Main.runRepl(main.kt:155)
	at org.partiql.cli.Main.main(main.kt:140)
ERROR!
```

It looks like the tutorial hasn't been updated since PartiQL v0.1.0 and doesn't account for a breaking v0.2.0 [change](https://github.com/partiql/partiql-lang-kotlin/releases/tag/v0.2.0-alpha) that requires an `ON` clause for `JOIN`s. I think the tutorial should use the most recent PartiQL releases and the tutorial queries should be fixed.

Also, should consider adding the tutorial queries as unit tests to make sure the tutorial is kept up to date.

### PR DESCRIPTION
**Pull request by alancai98**
*Issue #, if available:* fixes #413

*Description of changes:*
- adds `ON` clause to erroring tutorial queries
- removes timing and redundant whitespace in tutorial query files
- fixes some typos (e.g. `particl.bat` -> `partiql.bat`)
- `tutorial.html` and `tutorial.pdf` were generated through the docker process outlined [here](https://github.com/partiql/partiql-lang-kotlin/tree/ce06bcf8203ea963596fc723e0bb7c5d1b422dfd/docs/Docker)
  - (these two files can mostly be ignored as their changes are documented in `tutorial.md`)
  - it seems like the docker process hasn't been run in a while hence the large diff for the two files
- `cli/archive/Tutorial/code/*`'s changes are the same as `docs/tutorial/code/*`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by almann — APPROVED** [review]
Overall looks good to me--one minor suggestion below, but otherwise :ship: it!

File: docs/tutorial/tutorial.md [hunk 1]
```diff
@@ -297,18 +297,15 @@ about unnesting collections.
...
+If it helps you to think in terms of `JOIN`, you may replace the comma
+with `JOIN`. That is, the following two queries are equivalent.
 
 +-----------------------------------+-----------------------------------+
 | ```sql                            | ```sql                            | 
 | SELECT e.name AS employeeName,    | SELECT e.name AS employeeName,    |
 |        p.name AS projectName      |        p.name AS projectName      |
 | FROM hr.employeesNest AS e,       | FROM hr.employeesNest AS e JOIN   |
-|      e.projects AS p              |      e.projects AS p              |
+|      e.projects AS p              |      e.projects AS p ON true      |
```
[the remark this answers is not in the document]
Comment (alancai98) [reply docs/tutorial/tutorial.md]: Yeah good point. Changed in the latest commit

File: cli/archive/Tutorial/tutorial.html [hunk 2]
```diff
@@ -500,8 +500,8 @@ <h3 id="unnesting-nested-collections-using-join">Unnesting Nested Collections Us
 <p>One may think that the <code>FROM</code> clause of the example executes, in a sense, a <code>JOIN</code> between employees and projects. If it helps you to think in terms of <code>JOIN</code>, you may replace the comma with <code>JOIN</code>. That is, the following two queries are equivalent.</p>
```
Comment (alancai98) [inline cli/archive/Tutorial/tutorial.html:500]: `tutorial.html` and `tutorial.pdf` were generated through the docker process outlined [here](https://github.com/partiql/partiql-lang-kotlin/tree/ce06bcf8203ea963596fc723e0bb7c5d1b422dfd/docs/Docker). These two files can mostly be ignored as their changes are documented in `tutorial.md`.

**Review by almann — APPROVED** [review]
:ship:

**Pull request merged**